server/svix-server/src/core/types.rs

```diff
@@ -166,7 +168,7 @@ pub trait BaseId: Deref<Target = String> {
 fn validate_limited_str(s: &str) -> std::result::Result<(), ValidationErrors> {
     const MAX_LENGTH: usize = 256;
     lazy_static! {
-        static ref RE: Regex = Regex::new(r"^[a-zA-Z0-9\-_.]+$").unwrap();
+        static ref RE: Regex = Regex::new(r"^[a-zA-Z0-9\-_.]+$").expect("Error compiling Regex");
```

Comment:
The idea was to add clarity, this doesn't really add clarity, it just restates what the error message will have anyway.
I think it should stay unwrap everywhere, and become expect only where it adds clarity (mainly main.rs, lib.rs, cfg.rs, and maybe db/mod.rs. Potentially other places too, such as worker.rs).

Reply:
Actually, given your other comment showing the shitty error messages in unwrap, I wonder whether it makes sense to keep these changes. I just hate the development burden for the little added benefit. What do you think?